src/main/resources/changesets/changelog-consent-103.0.xml

```diff
@@ -0,0 +1,28 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="103.0" author="connorlbark">
+        <createTable tableName="user_file">
+            <column name="user_file_id" autoIncrement="true" type="int">
+                <constraints primaryKey="true"/>
+            </column>
+            <!-- Could be: Reference ID, Dataset ID column, etc. -->
+            <column name="entity_id" type="text"/>
+            <column name="file_name" type="text" >
+                <constraints nullable="false"/>
+            </column>
+            <column name="bucket_name" type="text">
+                <constraints nullable="false"/>
+            </column>
+            <column name="blob_name" type="text">
```

Comment:
Should this really just be blobId?

Reply:
right, that's why I didn't call it blob_id; I saved them separately and called it `blob_name` to make it less ambigious given [the `BlobId` class](https://googleapis.dev/java/google-cloud-clients/0.97.0-alpha/com/google/cloud/storage/BlobId.html), which takes in `BlobId.of(bucketName, blobName)` and is used by GCSService. however, it seems that this naming is confusing, and if we're going to remove the bucket since it is static, then I can rename it to something a bit clearer.  I'm still not convinced on `blob_id` since it is a loaded term.

Reply:
ohhh actually this documentation is out of date. I think it make make sense to instead save everything as a gsUtilUri: https://cloud.google.com/java/docs/reference/google-cloud-storage/latest/com.google.cloud.storage.BlobId#inherited-members . I didn't see this function in the old documentation. This way, there's only one string to keep track of. thus, it could be saved as `gcs_file_uri`, and hopefully be a lot less confusing.